### ISSUE TITLE
Logrotate not working with 1.1.5-3

### ISSUE THREAD

**Issue by Stephan3555** [graphite-project/docker-graphite-statsd#81]
Good morning,

logrotate seems not to work with the Docker Image 1.1.5-3. I even can not start the logrotate manually because I cannot find the service.

Under /etc/logrotate.d there is a logrotate file present.

**Comment by Stephan3555** [comment]
Thanks for your fast reply @deniszh !

One more small thing. At your docker hub repo you specified the base image as Phusion. Took me some time to realize that its actually alpine :-)

https://hub.docker.com/r/graphiteapp/graphite-statsd

**Comment by piotr1212** [comment]
I the "docker" way to do logging is to send logs to stdout and let docker daemon handle it.


> Ideally, applications log to stdout/stderr, and Docker sends those logs to the configured logging destination. 

from [logging-best-practices](https://success.docker.com/article/logging-best-practices)

**Comment by deniszh** [comment]
@Stephan3555 : yes, Readme on hub should be updated, will do that too.

@piotr1212 : You're right, but I want to give people a choice.

**Comment by Stephan3555** [comment]
Thanks @deniszh

**Comment by deniszh** [comment]
logrotate package was added to image. By default it is running daily by built-in /etc/periodic/daily/logrotate script
Will make a new release.

**Comment by deniszh** [comment]
Released in v1.1.5-4

**Comment by Stephan3555** [comment]
Hi @deniszh,

I was monitoring the logrotate in the new 1.1.5-4 version over the weekend. Unfortunately its still not working. The logrotate service is available now, but it seems its not getting triggered.

![image](https://user-images.githubusercontent.com/2537072/51467367-af865280-1d6c-11e9-8f77-b58df005b918.png)

If I run `logrotate -d /etc/logrotate.conf` the output is following:

```
logrotate -d /etc/logrotate.conf
WARNING: logrotate in debug mode does nothing except printing debug messages!  Consider using verbose mode (-v) instead if this is not what you want.

reading config file /etc/logrotate.conf
including /etc/logrotate.d
reading config file acpid
reading config file graphite-statsd
reading config file nginx
error: nginx:1 duplicate log entry for /var/log/nginx/access.log
error: found error in file nginx, skipping
reading config file redis
Reading state from file: /var/lib/logrotate.status
Allocating hash table for state file, size 64 entries

Handling 5 logs

rotating pattern: /var/log/messages  weekly (4 rotations)
empty log files are rotated, old logs are removed
considering log /var/log/messages
error: stat of /var/log/messages failed: No such file or directory
Creating new state

rotating pattern: /var/log/acpid.log  weekly (4 rotations)
empty log files are not rotated, old logs are removed
considering log /var/log/acpid.log
  log /var/log/acpid.log does not exist -- skipping
Creating new state
not running postrotate script, since no logs were rotated

rotating pattern: /var/log/*.log /var/log/*/*.log  52428800 bytes (10 rotations)
empty log files are not rotated, old logs are removed
considering log /var/log/carbon-aggregator.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/carbon.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log needs rotating
considering log /var/log/gunicorn.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/statsd.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/graphite/cache.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/graphite/exception.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/graphite/info.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/graphite/rendering.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/nginx/access.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
considering log /var/log/nginx/error.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log does not need rotating (log size is below the 'size' threshold)
rotating log /var/log/carbon.log, log->rotateCount is 10
dateext suffix '-20190121'
glob pattern '-[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]'
glob finding logs to compress failed
glob finding old rotated logs failed
copying /var/log/carbon.log to /var/log/carbon.log-20190121
truncating /var/log/carbon.log

rotating pattern: /var/log/nginx/*.log  weekly (4 rotations)
empty log files are rotated, old logs are removed
No logs found. Rotation not needed.

rotating pattern: /var/log/redis/*.log  weekly (4 rotations)
empty log files are not rotated, old logs are removed
considering log /var/log/redis/*.log
  log /var/log/redis/*.log does not exist -- skipping
Creating new state
```

By the way the nginx conf seems to have an syntax error.

```
considering log /var/log/carbon.log
Creating new state
  Now: 2019-01-21 10:11
  Last rotated at 2019-01-21 10:00
  log needs rotating

...

rotating log /var/log/carbon.log, log->rotateCount is 10
dateext suffix '-20190121'
glob pattern '-[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]'
glob finding logs to compress failed
glob finding old rotated logs failed
copying /var/log/carbon.log to /var/log/carbon.log-20190121
truncating /var/log/carbon.log
```
Logrotate stated that it was already rotated today, but as you can see on the screenshot above it was not.

if I run the logrotate without the '-d' flag (debug mode where changes are not applied):

![image](https://user-images.githubusercontent.com/2537072/51468175-97afce00-1d6e-11e9-9056-52713f166b7e.png)

The rotation of the file was successful (without the nginx logs).
For me it seems like that the servie that runs `/etc/periodic/daily/logrotate` is not working.

Greetings,
Stephan
